larray/tests/test_session.py

```diff
@@ -5,9 +5,8 @@
 import numpy as np
 import pytest
 
-from larray import Session, Axis, LArray, ndrange, isnan, larray_equal
 from larray.tests.common import assert_array_nan_equal, abspath
-from larray.util.misc import pickle
```

Comment:
I do not think pickle should be part of the public API so it should be imported from the sub module (and we should prevent it from ending up in larray top-level namespace